kickoff/static/treestatus.js

```diff
@@ -0,0 +1,49 @@
+function _isTreeForThunderbird(treeName) {
+    return treeName.indexOf('comm') > -1;
+}
+
+function getTreeStatusUrl(branchName) {
+    var treeName = branchName.split('/')[1];
+    if (_isTreeForThunderbird(treeName)) {
+        treeName += '-thunderbird';
+    }
+
+    return 'https://treestatus.mozilla.org/' + treeName + '?format=json';
```

Comment:
Good idea. However, I'd prefer to keep the plain URLs in the tests. If we didn't, we might make a nit in the config file (for instance: `https://trestatus.mozilla.org/`), and no test will break.